src/Chanterelle.purs

```diff
@@ -13,15 +13,19 @@ import Chanterelle.Project (loadProject)
 import Control.Monad.Error.Class (try)
 import Data.Either (Either(..))
 import Data.Generic.Rep (class Generic)
-import Data.Generic.Rep.Show (genericShow)
+import Data.Show.Generic (genericShow)
 import Effect.Aff (Aff)
 import Effect.Class (liftEffect)
 import Node.Path (resolve)
 import Node.Process (cwd, exit)
 
 
-data SelectCLI a b = SelectCLI a 
+data SelectCLI :: forall k. Type -> k -> Type
```

Comment:
are these kind annotations required? I don't think we need them for such simple types if they are optional